fs/copy_solaris.go

```diff
@@ -0,0 +1,63 @@
+package fs
```

Comment:
the difference is that darwin and freebsd define stat_t fields like Atimespec,
where as linux and solaris define stat_t fields like Atim.  i'll create accessor functions to hide these platform differences and then i can use copy_unix.go on solaris.